spec/spec_helper.rb

```diff
@@ -139,6 +139,7 @@ module Shell
   config.filter_run_excluding :not_supported_on_gce => true if gce?
   config.filter_run_excluding :not_supported_on_nano => true if windows_nano_server?
   config.filter_run_excluding :win2k3_only => true unless windows_win2k3?
+  config.filter_run_excluding :windows2012r2_only => true unless windows_2012r2?
```

Comment:
I would name this filter `win2012r2_only` to match the above filter's pattern, or alternatively `windows_2012r2_only` to match the below filter, but not this pattern in between.